core/variant.h

```diff
@@ -141,6 +145,7 @@ class Variant {
 		Basis *_basis;
 		Transform *_transform;
 		void *_ptr; //generic pointer
+		const void* _const_ptr;
```

Comment:
Is there any difference between the old `_ptr` and the new `_const_ptr`, given that they are part of a union?

Reply:
yes, one is const the other isn't, it's a small and somewhat irrelevant addition, you could as easily just const_cast but given it's a union and you don't pay for it I preferred this method to a cast.